utils/githooks/pre-commit.d/10-update-copyright

```diff
@@ -39,20 +45,16 @@ targets=(
     '.env'
 )
 
+if [ -z "$files" ]; then
+    files=$(git diff "$TARGET" --cached --diff-filter=AM --name-only -- "${targets[@]}")
```

Comment:
What are the ramifications of removing the `...` from target being diffed?

Reply:
I tried to explain here
https://github.com/daos-stack/daos/pull/14214#discussion_r1581329370

TL;DR the `...` seems to only include changes that are not staged, or that are already committed.

By example
When a single file is modified and added:
```
$ git add Jenkinsfile
$ git status
Changes to be committed:
        modified:   Jenkinsfile

# Wrong because it doesn't included the staged file
$ git diff origin/master... --name-only
# empty!

# Correct in this case
$ git diff origin/master --name-only
Jenkinsfile
```

When one file is modified and added, and another file is modified but not added
```
$ git status
Changes to be committed:
        modified:   Jenkinsfile

Changes not staged for commit:
        modified:   utils/rpms/daos.spec

# Wrong because it doesn't include either file
$ git diff origin/master... --name-only
# empty!

# Technically wrong because daos.spec is not staged for commit.
# I.e. don't run commit hooks against files that are modified but not going to be committed.
$ git diff origin/master --name-only
Jenkinsfile
utils/rpms/daos.spec

# --cached includes just the staged files.
$ git diff origin/master --name-only --cached
Jenkinsfile
```